Bio/.flake8

```diff
@@ -35,4 +35,10 @@ ignore =
     # ================================================
     # C812	missing trailing comma
     # C815	missing trailing comma in Python 3.5+
-    C812,C815
+    C812,C815,
+    # ================================================
+    # flake8 whitespace: W50#
+    # ================================================
+    # W503	Line break before binary operator
+    # PEP8 is now in favour of line-breaks before binary operators
+    W503
```

Comment:
Can you put this at line 10 of the file with the other flake8 built in codes please?

Reply:
Just add ```,W501,``` to line 10? (and also in the other ```.flake8``` files, of course)

Reply:
That'd be fine, thanks.

Looking at this afresh, we could list the non-default ignores explicitly (as they make be good targets for future enforcement).